src/app/components/Item/ItemTableRow.jsx

```diff
@@ -42,10 +42,10 @@ const ItemTableRow = ({ item, bibId, getRecord, searchKeywords }) => {
 
   return (
     <tr className={item.availability}>
-      <td>{item.location}</td>
-      <td>{itemCallNumber}</td>
-      <td>{itemRequestBtn}</td>
-      <td>{item.accessMessage.prefLabel}</td>
+      <td>{item.location || ' '}</td>
+      <td>{itemCallNumber || ' '}</td>
```

Comment:
I don't think you need to do this here. If there's no `item.callNumber`, it will be `` anyways. See line 39 -41